README.md

```diff
@@ -0,0 +1,85 @@
+## Table of Contents
+- [About](#about)
+- [How It Works](#how-it-works)
+- [Getting started](#getting-started)
+    - [Requirements](#requirements)
+    - [Installation](#installation)
+    - [Test config files](#test-config-files)
+    - [Testing Quick Starts](#installation-and-quick-start)
+
+
+## About
+Zathras is a test automation framework for use with bare metal, cloud, and virtualized systems. Tests are defined by wrappers - external, self-contained scripts that enable easy extension of Zathras's functionality. 
+
+Currently, Zathras offers the following test wrappers:
```

Comment:
Change currently to be 
As of 09/11/204
That way there is a time date present, and if a test is added it does not automatically cause problems.
Also add a line something like this.
To see the current available tests the version you are running, and the latest version issue the following command
./burden --test_version_check